src/test/emwasm_compile_known_gcc_test_failures.txt

```diff
@@ -69,6 +69,8 @@
 	20030913-1.c # glob multiply defined in wasm but not asm2wasm
         960218-1.c # glob multiply defined in wasm but not asm2wasm
 	pr17377.c # __builtin_return_address
+        20030323-1.c # __builtin_return_address
+        20030811-1.c # __builtin_return_address
```

Comment:
Nit: Line 70 seems to be misaligned, so I guess it's better to be consistent with other lines

Reply:
😳 for some reason this entire file is indented with tabs...😭

Reply:
Heresy! 😱